### PR TITLE
KoreanIM, add recipe

### PR DESCRIPTION
**Pull request by terrayi**

### CODE REVIEW

File: app-i18n/koreanim/koreanim-0.0.1.recipe [hunk 1]
```diff
@@ -0,0 +1,46 @@
+SUMMARY="Korean Input Method for BeOS/Haiku"
+DESCRIPTION="Korean Input Method for BeOS/Haiku using libhangul"
+HOMEPAGE="https://github.com/terrayi/KoreanIM"
+COPYRIGHT="2014 Terra Yi"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/terrayi/KoreanIM/archive/refs/tags/v0.0.1.zip"
+SOURCE_DIR="KoreanIM-$portVersion"
+CHECKSUM_SHA256="3af306a4bb5db591b60105e0f98e24f7956a748abe7170e0cf7760ffbf2bbd6b"
```
Comment (Begasus) [inline app-i18n/koreanim/koreanim-0.0.1.recipe:9]: SOURCE_DIR after CHECKSUM_SHA256 (check guidelines)

File: app-i18n/koreanim/koreanim-0.0.1.recipe [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+PATCHES="koreanim-0.0.1.patchset"
+
+ARCHITECTURES="all"
+
+PROVIDES="
+	KoreanIM = $portVersion
```
Comment (Begasus) [inline app-i18n/koreanim/koreanim-0.0.1.recipe:15]: Lowercase here, the name should reflect the recipes name:
```diff
- 	KoreanIM = $portVersion
+ 	koreanim = $portVersion
```

**Comment by Begasus** [comment]
Maybe move this to `haiku-apps`, if you search for `$addOnsDir` in the repository you'll find other input servers there also?
https://github.com/haikuports/haikuports/search?p=2&q=%24addOnsDir

File: app-i18n/koreanim/koreanim-0.0.1.recipe (hunk 1 above)
Comment (terrayi) [reply app-i18n/koreanim/koreanim-0.0.1.recipe:9]: That is wha I have missed! I have only looked at BUILD_* etc but not those meta info(?)

**Comment by terrayi** [comment]
> Maybe move this to `haiku-apps`, if you search for `$addOnsDir` in the repository you'll find other input servers there also?
> 
> https://github.com/haikuports/haikuports/search?p=2&q=%24addOnsDir

However, other input methods like canna and becjk and etc are under app-i18n directory. I guess this may be better resided under same directory with other input methods.

**Comment by Begasus** [comment]
> > Maybe move this to `haiku-apps`, if you search for `$addOnsDir` in the repository you'll find other input servers there also?
> > https://github.com/haikuports/haikuports/search?p=2&q=%24addOnsDir
> 
> However, other input methods like canna and becjk and etc are under app-i18n directory. I guess this may be better resided under same directory with other input methods.

Fair enough ;) didn't look any further, let's merge and see if buildmaster is Ok with this :)

**Comment by terrayi** [comment]
> Thanks for your contribution! 👍

Awesome! Thank you for prompt reviews and comments.

**Pull request merged**